.github/workflows/update-moon-landing.yml

```diff
@@ -0,0 +1,41 @@
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "js/**"
+
+jobs:
+  createPullRequest:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          repository: huggingface/moon-landing
+          ref: "main"
+          token: ${{ secrets.MOONLANDING_PUSH }}
+          path: "moon-landing"
+          submodules: true
+
+      - name: Make changes to pull request
+        run: cd moon-landing/server/hub-docs && git checkout origin/main
```

Comment:
should it checkout the exact commit-sha that's been triggering this action? otherwise race condition when two commits are triggered rapidly

(maybe overkill)

Reply:
I tried first with `git checkout ${{ github.sha }}`, didn't work for some reason (maybe need a fetch first)

Given my other comment (only one PR for all changes) I think it's ok to not take the exact commit